### ISSUE TITLE
make header files a separate repo

### ISSUE THREAD

**Issue by nshaheed** [ccrma/chugins#66]
Suggested by @celestebetancur 

In the context of #64, this makes a lot of sense - it serves as a single point of truth that you can pull from rather than the current policy of copy+paste the headers from chuck-core. That's fine if the assumption is that chugin repo is the single source of truth, but if we move to the chugin repo having stricter standards and encouraging people to make their own repos otherwise, we need a more central way to unify headers.

**Comment by spencersalazar** [comment]
makes sense -- `chuck/include` can simply point to the main chuck repo as a submodule
